src/Tickets/Emails/Admin/Preview_Modal.php

```diff
@@ -88,34 +88,30 @@ public function get_modal_args( $args = [] ): array {
 				'tribe-modal__title--emails-preview',
 			],
 		];
-
-		return wp_parse_args( $args, $default_args );
 	}
 
 	/**
 	 * Get the default modal contents.
 	 *
 	 * @since 5.5.7
 	 *
-	 * @param array $args Override default args by sending them in the `$args`.
-	 *
 	 * @return string The modal content.
 	 */
-	public function get_modal_content( $args = [] ): string {
-		/** @var Tribe__Tickets__Editor__Template $template */
+	public function get_modal_content(): string {
+		/** @var \Tribe__Tickets__Editor__Template $template */
 		$template = tribe( 'tickets.editor.template' );
 
 		$content = $template->template( 'v2/components/loader/loader', [], false );
 
-		$args = $this->get_modal_args( $args );
+		$args = $this->get_modal_args();
 
 		$dialog_view = tribe( 'dialog.view' );
 
 		ob_start();
 		$dialog_view->render_modal( $content, $args, static::$modal_id );
 		$modal_content = ob_get_clean();
 
-		$modal  = '<div class="tribe-common event-tickets">';
+		$modal = '<div class="tribe-common event-tickets">';
 		$modal .= '<span id="' . esc_attr( static::$modal_target ) . '"></span>';
 		$modal .= $modal_content;
 		$modal .= '</div>';
```

Comment:
GH is being a jerk about suggestions 😮‍💨 so:
instead of `$modal_content = ob_get_clean();`
return `ob_get_clean();` at the end and avoid all the variable assignment/concat